display.go

```diff
@@ -177,6 +182,11 @@ func printVPCs(vpcs []VPCSorted) {
 					)
 					for _, iface := range interfaceEndpoint.Interfaces {
 						//an endpoint can have multiple interfaces in multiple subnets, we only want to display whats relevant to the subnet
+						if Config.HideIP {
+							iface.MAC = ""
```

Comment:
Instead of printing nothing, maybe '<hidden mac>' or something?

Reply:
how about "xxx.xxx.xxx.xxx" and "xx:xx:xx:xx:xx:xx"  and "xxx.xxx.xxx.xxx/xx" to show that they were ip addresses, macs, and cidrs, but that they have been expunged?

Reply:
Perfect. Keep in mind ipv6 (not sure how that shows in your tool if at all)